README.md

```diff
@@ -11,12 +11,16 @@ Open and save 3D image stacks as animated gifs
 
 You find the menus for opening and saving animated gifs in the `Tools > File Import/Export` menu:
 
-![img.png](https://github.com/haesleinhuepf/napari-animated-gif-io/raw/main/docs/screenshot.png)
+![menu.png](docs/new-screenshot.png)
```

Comment:
Ah, I didn't know there was a reason behind it and changed it so that the new images/gifs from my fork would show up. I understand that I should then use the absolute URL to the original repo right?